apps/web/screens/UniversitySearch/LandingPage.tsx

```diff
@@ -59,13 +59,15 @@ const LandingPage: Screen<LandingPageProps> = ({
   useEffect(() => {
     const newArray = [...universities]
     newArray.sort((x, y) => {
-      const titleX = x.contentfulTitle || ''
-      const titleY = y.contentfulTitle || ''
-      return titleX.localeCompare(titleY)
+      const titleX =
+        locale === 'is' ? x.contentfulTitle || '' : x.contentfulTitleEn || ''
+      const titleY =
+        locale === 'is' ? y.contentfulTitle || '' : y.contentfulTitleEn || ''
+      return titleX.localeCompare(titleY, locale)
```

Comment:
Perhaps using the sortAlpha function instead would help out with having "á" after "a" and so forth (not sure though if that even is a problem but just wanted to mention it

Reply:
Has not been a problem for this case,  but I didn't know we had a sortAlpha in our utils, good to know!